tools/EteriTools/CMFinder/cmFinder.py

```diff
@@ -0,0 +1,58 @@
+import glob, os, sys
+import shutil
+from shutil import copyfile
+import subprocess
+from os import system
+
+def sh(script):
+    system("bash -c '%s'" % script)
+
+
+#print sh("pwd")
+
+
+#input_dir = sys.argv[1]
+model_tree_stk = sys.argv[1]
+cmfinder_fa = sys.argv[2]
+path = sys.argv[3]
+
+gapCmd = ""
+gapVal = ""
+if len(sys.argv) > 4:
+    gapCmd = sys.argv[4]
+    gapVal = sys.argv[5]
+
+
+#sh("mkdir  CLUSTER")
```

Comment:
Can you clean this up please and use flake8 to find some errors and best-practices for all your python files.